easybuild/easyconfigs/f/fastStructure/fastStructure-1.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -34,8 +35,10 @@ cmds_map = [('.*', 'cd vars && python setup.py build_ext --inplace && cd .. && p
 files_to_copy = ['*']
 
 postinstallcmds = [
-    'echo "#!/bin/env python" | cat - %(installdir)s/structure.py > temp && mv temp %(installdir)s/structure.py',
-    'chmod +x %(installdir)s/structure.py'
+    'for pyfile in %(installdir)s/*.py; do ' +
+    'echo "#!/bin/env python" | cat - ${pyfile} > temp && mv temp ${pyfile}; ' +
+    'chmod +x ${pyfile}; ' +
+    'done'
```

Comment:
Hmm, this may be a bit too much "abuse" in an easyconfig file...

Can't we figure out a better way to handle this cleanly? One way would be to create a custom easyblock for `fastStructure`...

Reply:
I created an easyblock for fastStructure:
https://github.com/easybuilders/easybuild-easyblocks/pull/1529